iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -661,7 +666,7 @@ public boolean isTimeTravelAllowed() {
   }
 
   @Override
-  public boolean isMetadataTableSupported() {
+  public boolean isTableIdentifierSupported() {
```

Comment:
not sure it's an expected rename. Metadata table is supported only in iceberg, but table id should be available for every table type

Reply:
I explained this in following comment.

Reply:
Changed this method to `isTableMetaRefSupported()`